tutorials/plot_morph_stc.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
+r"""
+===========================================
+Morphing source estimates using SourceMorph
+===========================================
+
+In this tutorial we will morph different kinds of source estimation results
+between individual subject spaces using :class:`mne.SourceMorph`.
+For group level statistical analyses subject specific results have to be mapped
+to a common space.
+
+We will use precomputed data and morph surface and volume source estimates to a
+common space. The common space of choice will be FreeSurfer's "fsaverage".
```

Comment:
For this particular example 'fsaverage' from FreeSurfer is used as reference space.

I like reference better than common space. (but whatever it is, it should be consistent with the docstrings)